json_reader.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"encoding/json"
+	"strings"
+)
+
+// jsonReader implements the sourceReader interface.
+type jsonReader struct{}
+
+// Unmarshal decodes the byte slice into a go readable format.
+// Unmarshal implements the sourceReader interface.
+func (*jsonReader) Unmarshal(bts []byte) (interface{}, error) {
+	jsonData := map[string]interface{}{}
```

Comment:
Was ist das für eine Annahme hier? Warum map[string]interface{} ? Besser einfach interface...

Reply:
I did this because the interface automatically points to `map[string]interface{}` if no type is present.

I changed it to `interface{}`.